hack/generate.sh

```diff
@@ -15,7 +15,7 @@ if [ "${IS_CONTAINER}" != "false" ]; then
   cd "${GOPATH}"/src/github.com/metal3-io/baremetal-operator
   export XDG_CACHE_HOME="/tmp/.cache"
 
-  INPUT_FILES="deploy/crds/*.yaml pkg/apis/metal3/v1alpha1/zz_generated.*.go"
+  INPUT_FILES="config/crd/bases/*.yaml apis/metal3.io/v1alpha1/zz_generated.*.go"
```

Comment:
Should we check the whole config folder ? some files might change outside of the crd folder ( like webhooks for example)

Reply:
Yes, good point.